### DISCUSSION TITLE
File manager should remember where you were when you edit a file and return you there

### DISCUSSION THREAD

**Discussion by nep** [ExpressionEngine/ExpressionEngine#4326] [Feature Requests]
### Suggest an idea for ExpressionEngine

**Is your feature request related to a problem? Please describe.**
It's very frustrating to be looking for at files, click to edit it or modify its meta data, and then be returned to the first page of the main file interface when what you want is to examine or change the next file.

**Describe the solution you'd like**
The current state of the File manager should be preserved when you click to edit or investigate a file, just the way it is done in the Entry manager.

**Describe alternatives you've considered**
Drupal :P

**Teachability, Documentation, Adoption, Migration Strategy**
I don't think there's any modifications needed to the documentation. This would be standard, unsurprising behavior without any training or explanation needed.

**Comment by intoeetive** [comment]
https://github.com/ExpressionEngine/ExpressionEngine/pull/4454
